app/src/main/java/ch/epfl/sweng/SDP/home/BattleLogActivity.java

```diff
@@ -33,11 +36,21 @@ protected void onCreate(Bundle savedInstanceState) {
                 .into((ImageView) findViewById(R.id.backgroundAnimation));
 
         TextView exitButton = findViewById(R.id.exitButton);
-        LayoutUtils.setExitListener(exitButton, this);
+        TextView emptyBattleLogText = findViewById(R.id.emptyBattleLogText);
+        LayoutUtils.setFadingExitListener(exitButton, this);
+
         exitButton.setTypeface(typeMuro);
+        emptyBattleLogText.setTypeface(typeMuro);
         ((TextView) findViewById(R.id.battleLogText)).setTypeface(typeMuro);
 
         fetchGameResults();
+
+        ScrollView scrollBattleLog = findViewById(R.id.scrollBattleLog);
```

Comment:
I think you should comment or extract method here to make this code more understandable